File: app/src/main/java/ptv/example/zoulinheng/androidutils/MyApplication.java
package ptv.example.zoulinheng.androidutils;

import android.app.Application;
import android.content.Context;
import android.content.Intent;

import androidx.multidex.MultiDex;

import ptv.example.zoulinheng.androidutils.download.DownLoadService;

public class MyApplication extends Application {
    private static MyApplication myApplication;

    public static MyApplication getInstance() {
        return myApplication;
    }

    @Override
    public void onCreate() {
        super.onCreate();
        myApplication = this;
        this.startService(new Intent(this, DownLoadService.class));
    }

    @Override
    protected void attachBaseContext(Context base) {
        MultiDex.install(this);
        super.attachBaseContext(base);
    }
}

File: app/src/main/java/ptv/example/zoulinheng/androidutils/MyApplication.kt
package ptv.example.zoulinheng.androidutils

import android.app.Application
import android.content.Context
import android.content.Intent
import androidx.multidex.MultiDex
import ptv.example.zoulinheng.androidutils.download.DownLoadService

class MyApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        startService(Intent(this, DownLoadService::class.java))
    }

    override fun attachBaseContext(base: Context) {
        MultiDex.install(this)
        super.attachBaseContext(base)
    }
}